lib/routes.json

```diff
@@ -10240,6 +10240,90 @@
         }
       },
       "url": ":url"
+    },
+    "getPreReceiveHook": {
+      "headers": {
+        "accept": "application/vnd.github.eye-scream-preview"
+      },
+      "method": "GET",
+      "params": {
+        "id": {
```

Comment:
please use `pre_receive_hook_id` instead of `id` as parameter to align with the documentation at https://developer.github.com/enterprise/2.14/v3/repos/pre_receive_hooks/. Update in `params: {...}` objects and the `url` string